web_src/js/features/contextPopup.js

```diff
@@ -20,14 +20,15 @@ function issuePopup(suburl, owner, repo, index, $element) {
     let labels = '';
     for (let i = 0; i < issue.labels.length; i++) {
       const label = issue.labels[i];
+      const labelName = emojify.replace(label.name);
       const red = parseInt(label.color.substring(0, 2), 16);
       const green = parseInt(label.color.substring(2, 4), 16);
       const blue = parseInt(label.color.substring(4, 6), 16);
       let color = '#ffffff';
       if ((red * 0.299 + green * 0.587 + blue * 0.114) > 125) {
         color = '#000000';
       }
-      labels += `<div class="ui label" style="color: ${color}; background-color:#${label.color};">${label.name}</div>`;
+      labels += `<div class="ui label has-emoji" style="color: ${color}; background-color:#${label.color};">${labelName}</div>`;
```

Comment:
```suggestion
      labels += `<div class="ui label" style="color: ${color}; background-color:#${label.color};">${labelName}</div>`;
```

No longer needed, it only works on page load.

Reply:
I thought that specify text inside. Done as you wish :)

Reply:
A more appropriate name would be `replace-emoji-tokens` or something.